go-controller/pkg/node/gateway_iptables.go

```diff
@@ -62,9 +62,11 @@ type iptRule struct {
 	protocol iptables.Protocol
 }
 
-func addIptRules(rules []iptRule) error {
-	var addErrors, err error
+func addIptRules(rules []iptRule, append bool) error {
+	addErrors := errors.New("")
+	var err error
```

Comment:
is there a reason for declaring `err` and `exists` here?

Reply:
So I think err was leftover from previous code - hopefully it wasn't a git blame me haha, and because we were declaring err globally and then redeclaring it inside the for loop again since exists wasn't declared there were some issues with err's scopes when used from different places in the loop.
I could just move both to be declared inside the loop if needed, just went with the flow...

Reply:
this got merged with the other PR, wasn't part of this PR...